### PR TITLE
React-Flux example

### PR DESCRIPTION
**Pull request by lelandrichardson**
I created an example using the [react-flux library](https://github.com/kjda/ReactFlux) which I think is a really great library that deserves more attention than it's getting!  Let me know if there is anything that I need to change in order to get the PR merged in.

BTW, this is a great repo. With all of the flux madness going about these days, it's nice to have these comparative examples!

### CODE REVIEW

**Comment by voronianski** [comment]
@lelandrichardson do you know why this library depends on old React - https://github.com/kjda/ReactFlux/blob/master/package.json#L30 ?

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,4 @@ alt/build/bundle.js
 mcfly/build/bundle.js
 marty/build/bundle.js
 flummox/build/bundle.js
```
Comment (voronianski) [inline .gitignore:13]: @lelandrichardson please add `bundle.js` to ignore list.

File: react-flux/js/actions/CartActions.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+var Flux = require('react-flux');
```
Comment (voronianski) [inline react-flux/js/actions/CartActions.js:3]: @lelandrichardson all `.js` files are missing `'use strict';` line at top, please add it - (https://github.com/voronianski/flux-comparison/blob/master/.jshintrc#L40).

**Comment by lelandrichardson** [comment]
@voronianski I'm not sure why react is listed as a dependency. it shouldn't be (there is no hard dependency on react).

I'll submit a PR to the react-flux repo to change it.

**Comment by lelandrichardson** [comment]
PR created: https://github.com/kjda/ReactFlux/pull/9

**Comment by lelandrichardson** [comment]
@voronianski the react dependency should now be removed from ReactFlux. Anything else I need to change?

Thanks.

**Comment by voronianski** [comment]
@lelandrichardson sorry, this doesn't work as expected.. 

Take a look on http://labs.voronianski.com/flux-comparison/ and compare with yours. This code is not working in your example - https://github.com/voronianski/flux-comparison/blob/master/common/components/ProductItem.jsx#L25

Another note regarding the name of this lib, "React-Flux" sounds a bit confusing to me :) what do you think?

**Comment by voronianski** [comment]
Closing this as out of branch. If you'd like to see it merge please fix the issue and make new PR.

**Pull request closed without merging**